geode-core/src/integrationTest/java/org/apache/geode/distributed/internal/membership/MembershipJUnitTest.java

```diff
@@ -452,65 +446,4 @@ public void testMulticastDiscoveryNotAllowed() {
       // expected
     }
   }
-
```

Comment:
was this replaced?

Reply:
These classes no longer subclass DistributionMessage, so their "process" messages were removed and that made these tests unnecessary.